src/utils/services/chainlink.service.ts

```diff
@@ -39,6 +39,7 @@ export class PriceFeedService {
         [AcceptedToken.USDT]: '0x5e37AF40A7A344ec9b03CCD34a250F3dA9a20B02',
         [AcceptedToken.CUSD]: '0xe38A27BE4E7d866327e09736F3C570F256FFd048',
         [AcceptedToken.CELO]: '0x0568fD19986748cEfF3301e55c0eb1E729E0Ab7e',
+        // No dedicated cEUR price feed available on Celo network - will fall back to default price of $1.00
```

Comment:
This is wrong, cEUR is an unchained representation of Euro which is definitely not same as $1

Reply:
https://docs.chain.link/data-feeds/price-feeds/addresses?page=2&testnetPage=2&network=celo


You can always check the feed addresses here